Shared/Samples/Show device location with NMEA data sources/ShowDeviceLocationWithNMEADataSourcesView.Model.swift

```diff
@@ -61,9 +51,9 @@ extension ShowDeviceLocationWithNMEADataSourcesView {
         
         /// A mock data source to read NMEA sentences from a local file, and generate
         /// mock NMEA data every fixed amount of time.
-        private let mockNMEADataSource = SimulatedNMEASentenceFeed(
-            nmeaSourceFile: Bundle.main.url(forResource: "Redlands", withExtension: "nmea")!,
-            speed: 1.5
+        private let mockNMEAFeed = FileNMEASentenceReader(
+            url: Bundle.main.url(forResource: "Redlands", withExtension: "nmea")!,
+            interval: 0.66
```

Comment:
2 things

1. The comment uses the term "data source", but I think that is overloaded in this context.
2. I think we should call the variable simply `sentenceReader`